### PR TITLE
feat: initial open source version of migration tool working with unix docker emulator

### PR DESCRIPTION
**Pull request by ppiwow-apay**

### CODE REVIEW

**Comment by starkpl** [comment]
Please add a CI workflow, similar to this:
https://github.com/allegro/cosmosdb-utils/blob/main/.github/workflows/Allegro.CosmosDb.BatchUtilities.ci.yml

**Comment by starkpl** [comment]
Please mention the utility in the main README:
https://github.com/allegro/cosmosdb-utils#utilities

File: src/Allegro.CosmosDb.Migrator/src/Allegro.CosmosDb.Migrator.Api/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "app": {
+    "name": "CosmosMigrator Service API"
+  },
+  "cosmos": {
+    "connectionString": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
```
Comment (starkpl) [inline src/Allegro.CosmosDb.Migrator/src/Allegro.CosmosDb.Migrator.Api/appsettings.json:6]: What KEY is this? Isn't it sensitive? Do we need to commit it?

File: src/Allegro.CosmosDb.Migrator/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Introduction 
+This project contains fully working service that enables real time as is migration between two CosmosDb containers.
+
+It contains:
+- migration api
+- migration statistics api
+
+# Getting Started
```
Comment (starkpl) [inline src/Allegro.CosmosDb.Migrator/README.md:8]: Maybe we should briefly describe here how to use it (copy the solution locally, set connection strings, run with script etc)

File: src/Allegro.CosmosDb.Migrator/scripts/build.cmd [hunk 3]
```diff
@@ -0,0 +1 @@
+dotnet build -c release
```
Comment (starkpl) [inline src/Allegro.CosmosDb.Migrator/scripts/build.cmd]: All these scripts will not work when run from the /scripts subdirectory. How exactly should they be run? Maybe some description is needed in README?
On Mac OS or linux you can probably run them like this:
```
cd src/Allegro.CosmosDb.Migrator
./scripts/build.sh
```
I'm not sure about windows

File: src/Allegro.CosmosDb.Migrator/src/Allegro.CosmosDb.Migrator.Api/appsettings.json (hunk 1 above)
Comment (ppiwow-apay) [reply src/Allegro.CosmosDb.Migrator/src/Allegro.CosmosDb.Migrator.Api/appsettings.json:6]: all emulator (i checked windows installer and unix docker) provide same key, so this is emulator key for people who wants to play locally. will add some note to readme

File: src/Allegro.CosmosDb.Migrator/README.md (hunk 2 above)
Comment (ppiwow-apay) [reply src/Allegro.CosmosDb.Migrator/README.md:8]: added some simple instruction

File: src/Allegro.CosmosDb.Migrator/scripts/build.cmd (hunk 3 above)
Comment (ppiwow-apay) [reply src/Allegro.CosmosDb.Migrator/scripts/build.cmd]: i'll delete them, they are useless

**Comment by ppiwow-apay** [comment]
> 

done

**Review by starkpl — APPROVED** [review]

**Pull request merged**